prow/bugzilla/client.go

```diff
@@ -92,6 +94,54 @@ func (c *client) GetBug(id int) (*Bug, error) {
 	return parsedResponse.Bugs[0], nil
 }
 
+func getClones(c Client, bug *Bug) ([]*Bug, error) {
+	clones := []*Bug{}
+	for _, dependentID := range bug.Blocks {
+		dependent, err := c.GetBug(dependentID)
+		if err != nil {
+			return nil, fmt.Errorf("Failed to get dependent bug #%d: %v", dependentID, err)
```

Comment:
Can we not return on the first error but instead aggregate the errors (via a `[]error` and `utilerrors.NewAggregate`) and just `continue` on err here?